extensions/librdkafka/PublishKafka.cpp

```diff
@@ -319,27 +320,29 @@ void PublishKafka::onTrigger(const std::shared_ptr<core::ProcessContext> &contex
 
   std::string client_id, brokers, topic;
 
+  std::unique_ptr<KafkaLease> lease = nullptr;
```

Comment:
This is purely stylistic, but the default constructor of `std::unique_ptr<T>` already initializes the unique_ptr to own nothing, so assigning a `nullptr` is superfluous.